### PR TITLE
Rename hiuvi -> Caspion

### PR DESCRIPTION
**Pull request by brafdlog**

### CODE REVIEW

File: src/backend/export/outputVendors/googleSheets/googleSheets.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import * as googleSheets from './googleSheetsInternalAPI';
 import { appendToSpreadsheet } from './googleSheetsInternalAPI';
 
 const GOOGLE_SHEETS_DATE_FORMAT = 'DD/MM/YYYY';
-const DEFAULT_SHEET_NAME = '_hiuvi';
+const DEFAULT_SHEET_NAME = '_caspion';
```
Comment (brafdlog) [inline src/backend/export/outputVendors/googleSheets/googleSheets.ts:13]: This is a breaking change for people who are already using the integration with google sheets, but I think it is worth doing it now and not leaving this old reference in the code.

Comment (baruchiro) [reply src/backend/export/outputVendors/googleSheets/googleSheets.ts:13]: Yes... but we will need to stop breaking the app someday 😉

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ jobs:
         script: |
           const releases = await github.repos.listReleases({
             owner: 'brafdlog',
-            repo: 'hiuvi'
+            repo: 'caspion'
```
Comment (baruchiro) [inline .github/workflows/release.yml:39]: You need to change the repo in Github

**Review by baruchiro — APPROVED** [review]
Just change the repo name in Github and we will see the checks are passed.

**Pull request merged**